tests/js/router/router-tests.js

```diff
@@ -1,17 +1,32 @@
 /* Test environment for the "router" grade and "wrapper" modules for common Express routers. */
 "use strict";
-var fluid        = fluid || require("infusion");
-var gpii         = fluid.registerNamespace("gpii");
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
 
-fluid.setLogging(true);
+// Load all of the components to be tested and our test cases
+require("../includes.js");
+require("./router-caseholder");
+require("./test-router-hello");
+require("./test-router-params");
+require("./test-router-reqview");
 
-var path         = require("path");
+var viewDir    = fluid.module.resolvePath("%gpii-express/tests/views");
+var contentDir = fluid.module.resolvePath("%gpii-express/tests/html");
```

Comment:
Support resolving this in the middleware grade too. We should get to the point where all grade definitions just consist of JSON.